devices/motorola-potter/kernel/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  mobile-nixos
+, stdenv
+, fetchFromGitHub
+, ...
+}:
+
+#
+# Note:
+# This kernel build is special, it supports both armv7l and aarch64.
+# This is because motorola ships an armv7l userspace from stock ROM.
+#
+# in local.nix:
+#  mobile.system.system = lib.mkForce "armv7l-linux";
+#
```

Comment:
Similarly, unless you have validated it works just as well for armv7l, remove this. (And `
stdenv.hostPlatform.parsed.cpu.name` use...)

Reply:
gone